niri-config/src/lib.rs

```diff
@@ -1635,6 +1635,12 @@ pub enum Action {
     FocusWindowBottom,
     FocusWindowDownOrTop,
     FocusWindowUpOrBottom,
+    FocusWindowOrWorkspaceOrMonitorUp(
+        #[knuffel(property(name = "skip-empty"), default = true)] bool,
```

Comment:
Can we do without skip_empty for now? Especially since it's unimplemented?..

Reply:
Hey, this is precisely the reason why the PR is in draft, I still don't know how to do the skip so I'm leaving it for later before having the PR ready